logpolicy/logpolicy.go

```diff
@@ -74,7 +74,9 @@ func getLogTarget() string {
 // logtail server, or the default.
 func LogHost() string {
 	if v := getLogTarget(); v != "" {
-		return v
+		if u, err := url.Parse(v); err == nil {
```

Comment:
Should we do this is in the `sync.Once` above?

Reply:
I considered it but there are like barely any callers and none performance sensitive so I erred on making a smaller diff

Reply:
But if you feel strongly, you/I can in a follow-up?

Reply:
Nah, just wanted to check 👍